### PR TITLE
Modified ScheduledInterviewDAO interface

### PR DESCRIPTION
**Pull request by GrantComm**
As I was working on the datastore dao, I realized I needed to change the return type of my create method and add a delete method for the future cancellation feature.

### CODE REVIEW

File: src/main/java/com/google/sps/data/ScheduledInterviewDao.java [hunk 1]
```diff
@@ -29,11 +28,12 @@
   // Returns a list of the ScheduledInterview objects that the user participates in.
   public List<ScheduledInterview> getForPerson(String email);
 
-  // Creates a ScheduledInterview entity.
-  public Entity create(ScheduledInterview scheduledInterview);
+  // Creates a ScheduledInterview entity and stores it in Datastore.
+  public void create(ScheduledInterview scheduledInterview);
 
   // Updates a ScheduledInterview.
   public void update(ScheduledInterview oldScheduledInterview);
```
Comment (apluscs) [inline src/main/java/com/google/sps/data/ScheduledInterviewDao.java]: This naming confuses me. Why is an `oldScheduledInterview` being put into datastore?

File: src/main/java/com/google/sps/data/ScheduledInterviewDao.java [hunk 2]
```diff
@@ -29,11 +28,12 @@
   // Returns a list of the ScheduledInterview objects that the user participates in.
   public List<ScheduledInterview> getForPerson(String email);
 
-  // Creates a ScheduledInterview entity.
-  public Entity create(ScheduledInterview scheduledInterview);
+  // Creates a ScheduledInterview entity and stores it in Datastore.
+  public void create(ScheduledInterview scheduledInterview);
```
Comment (apluscs) [inline src/main/java/com/google/sps/data/ScheduledInterviewDao.java:32]: Can you change create to put to match Datastore syntax?
Also less confusing for the Autovalue create()

**Review by apluscs — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/sps/data/ScheduledInterviewDao.java [hunk 3]
```diff
@@ -29,11 +28,12 @@
   // Returns a list of the ScheduledInterview objects that the user participates in.
   public List<ScheduledInterview> getForPerson(String email);
 
-  // Creates a ScheduledInterview entity.
-  public Entity create(ScheduledInterview scheduledInterview);
+  // Creates a ScheduledInterview entity and stores it in Datastore.
```
Comment (apluscs) [inline src/main/java/com/google/sps/data/ScheduledInterviewDao.java]: Since this is the interface, Datastore shouldn't be mentioned.

**Review by apluscs — CHANGES_REQUESTED** [review]
One nit.

**Review by apluscs — APPROVED** [review]

**Review by meagancheese — APPROVED** [review]
LGTM

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**